includes/net-standard-table.md

```diff
@@ -16,7 +16,7 @@
 - The columns represent .NET Standard versions. Each header cell is a link to a document that shows which APIs got added in that version of .NET Standard.
 - The rows represent the different .NET implementations.
 - The version number in each cell indicates the *minimum* version of the implementation you'll need in order to target that .NET Standard version.
-- For an interactive table, see [.NET Standard versions](https://immo.landwerth.net/netstandard-versions/#).
+- For an interactive table, see [.NET Standard versions](https://dotnet.microsoft.com/platform/dotnet-standard).
```

Comment:
You can use https://dotnet.microsoft.com/platform/dotnet-standard#versions if you want to send folks scrolled to the table

Reply:
Good catch. Fixed!